file-formats/clas/type/zif_oo_aff_clas_v1.intf.abap

```diff
@@ -39,33 +33,18 @@ INTERFACE zif_oo_aff_clas_v1 PUBLIC.
       "! <p class="shorttext">Business class</p>
       "! Business class
       business_class                 TYPE ty_category VALUE '50',
-      "! <p class="shorttext">Business class (static components)</p>
-      "! Business class interface for static components
-      business_class_intf_for_static TYPE ty_category VALUE '51',
-      "! <p class="shorttext">Business class (instance-dep. components)</p>
-      "! Business class interface for instance-dependent components
-      business_class_intf_for_instan TYPE ty_category VALUE '52',
       "! <p class="shorttext">BSP application class</p>
       "! BSP application class
       bsp_application_class          TYPE ty_category VALUE '60',
       "! <p class="shorttext">Basis class for BSP element handlers</p>
       "! Basis class for BSP element handlers
       basis_class_bsp_element_hdlr   TYPE ty_category VALUE '70',
-      "! <p class="shorttext">Web Dynpro runtime object</p>
-      "! Web Dynpro runtime object
-      web_dynpro_runtime_object      TYPE ty_category VALUE '80',
```

Comment:
We should add this. This is valid at least for generated web dynpro runtime classes

Reply:
Done